cosmos/proto/polaris/evm/v1alpha1/tx.proto

```diff
@@ -44,9 +44,13 @@ service MsgService {
 
 // EthTransactionRequest encapsulates an Ethereum transaction as an SDK message.
 message EthTransactionRequest {
+  option (cosmos.msg.v1.signer) = "from";
   option (gogoproto.goproto_getters) = false;
   // data is inner transaction data of the Ethereum transaction
   bytes data = 1;
+
+  // from is the sender of the transaction
+  string from = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
@tac0turtle is there a way to avoid this? I hate that it's forcing me to use bech32 with the eth transaction